src/services/user/getDashboardVars.ts

```diff
@@ -0,0 +1,23 @@
+import { AES, enc } from 'crypto-js';
+import { Settings, ISettings, IVariables } from 'db-models-nc';
+import config from '../../config';
+
+const { ENCRYPTION_SECRET_KEY } = config.server;
+
+type IGetDashboardVars = () => Promise<IVariables>
+
+const getDashboardVars: IGetDashboardVars = async () => {
+  const oldVars = (await Settings.findOne({ where: { name: 'variables' } })) as ISettings;
+  const oldVarsValue = JSON.parse(oldVars.value as string) as IVariables;
+
```

Comment:
Same as above
No need to parse it as its type JSON in DataTypes so Sequelize will handle parsing it
```suggestion
  const oldVarsValue = oldVars.value as IVariables;
```